schemas/context/profile-contact-crm.schema.json

```diff
@@ -0,0 +1,154 @@
+          "type": "string"
+        },
+        "xdm:emailBounced": {
+          "title": "Email Bounced Indicator",
+          "description": "If bounce management is activated and an email is sent to a contact, indicates whether the email bounced (true) or not (false).",
+          "type": "boolean"
+        },
+        "xdm:leadSource": {
+          "title": "Lead Source",
+          "description": "Field recording what source the lead originated from.",
+          "type": "string"
+        },
+        "xdm:contactDescription": {
+          "title": "Contact Description",
+          "description": "Contact description.",
+          "type": "string"
+        },
+        "xdm:contactStatus": {
+          "title": "Contact Status",
+          "description": "Current status of the contact.",
+          "type": "string"
+        },
+        "xdm:contactStatusDetail": {
+          "title": "Contact Status Detail",
+          "description": "Additional Details of the current status of the contact.",
+          "type": "string"
+        },
+        "xdm:contactType": {
+          "title": "Contact Type",
+          "description": "Type of the contact.",
+          "type": "string"
+        },
+        "xdm:userID": {
+          "title": "User Id",
+          "description": "User id.",
+          "type": "string"
+        },
+        "xdm:originalReferrer": {
+          "title": "Original Referrer",
+          "description": "The original referrer is typically the page that leads to a form fill out.",
+          "type": "string"
+        },
+        "xdm:originalSearchEngine": {
+          "title": "Original Search Engine",
+          "description": "Search engine that the original referrer used.",
+          "type": "string"
+        },
+        "xdm:originalSearchPhrase": {
+          "title": "Original Search Phrase",
+          "description": "Search phrase used by the original referrer.",
+          "type": "string"
+        },
+        "xdm:originalSourceInfo": {
+          "title": "Original Source Info",
+          "description": "Original source info.",
+          "type": "string"
+        },
+        "xdm:originalSourceType": {
+          "title": "Original Source Type",
+          "description": "Original ource type.",
+          "type": "string"
+        },
+        "xdm:timezone": {
+          "title": "Timezone",
+          "description": "Timezone of the contact's location.",
+          "type": "string"
+        },
+        "xdm:emailOptOutReason": {
+          "title": "Email Opt Out Reason",
+          "description": "Reason for the contact opting out of recieving any emails if available.",
+          "type": "string"
+        },
+        "xdm:phoneOptOutReason": {
+          "title": "Phone Opt Out Reason",
+          "description": "Reason for the contact opting out of recieving any phone calls if available.",
+          "type": "string"
+        }
+      }
+    }
+  },
+  "allOf": [
```

Comment:
Include all the data types/mixins which are required to build the complete profile-contact-CRM even if the customer is not with Marketo.

Reply:
Good point. Will do